pytest/tests/stress/stress.py

```diff
@@ -558,16 +571,34 @@ def doit(s, n, N, k, monkeys, timeout):
          ["block_producer_kickout_threshold", 10],
          ["chunk_producer_kickout_threshold", 10]], local_config_changes)
 
+    monkey_names = [x.__name__ for x in monkeys]
+    proxy = None
+    logging.info(monkey_names)
+    if 'monkey_local_network' in monkey_names or 'monkey_global_network' in monkey_names:
+        assert config['local'], 'Network stress operations only work on local nodes'
+        reject_list = RejectListProxy.create_reject_list(1)
+        proxy = RejectListProxy(reject_list)
+        expect_network_issues()
+        block_timeout += 40
+        balances_timeout += 20
```

Comment:
Also added 20s extra to wait for transactions to apply when the network is being messed up with.